sbndqm/dqmAnalysis/CRT/SBND/BernCRTdqmSBND_module.cc

```diff
@@ -136,175 +150,154 @@ sbndaq::BernCRTdqmSBND::~BernCRTdqmSBND()
+      continue;}
 
-      artdaq::ContainerFragment cont_frag(frag);
+    artdaq::ContainerFragment cont_frag(frag);
     
-      unsigned int const fragid = frag.fragmentID();
-      std::string fragment_id = std::to_string(fragid);
+    unsigned int const fragid = frag.fragmentID();
+    std::string fragment_id = std::to_string(fragid);
 
-      //get frag count
-      uint64_t frag_count = cont_frag.block_count();
-      //get zero rate
-      uint64_t nzero = 0;
-      if (frag_count == 0) { nzero = 1; }
+    //get frag count
+    uint64_t frag_count = cont_frag.block_count();
+    //get zero rate
+    uint64_t nzero = 0;
+    if (frag_count == 0) { nzero = 1; }
 
-      std::string group_name = "unknown_cont_frag";
+    std::string group_name = "unknown_cont_frag";
 
-      if (cont_frag.fragment_type() == sbndaq::detail::FragmentType::CAENV1730) {group_name = "PMT_cont_frag";}
-      else if (cont_frag.fragment_type() == sbndaq::detail::FragmentType::BERNCRTV2) {group_name = "CRT_cont_frag";} //this one is relevant for us
-	//print out arguments of the sendMetric line
-	//i.e. print out fragment_id to match to fcl
-	if (debug) std::cout<<"fragment_id: "<<fragment_id<<std::endl;
+    if (cont_frag.fragment_type() == sbndaq::detail::FragmentType::CAENV1730) {group_name = "PMT_cont_frag";}
+    else if (cont_frag.fragment_type() == sbndaq::detail::FragmentType::BERNCRTV2) {group_name = "CRT_cont_frag";}
+    if (fDebug) std::cout<<"fragment_id: "<<fragment_id<<std::endl;
 
-      sbndaq::sendMetric(group_name, fragment_id, "frag_count", frag_count, 0, artdaq::MetricMode::Average);
-      sbndaq::sendMetric(group_name, fragment_id, "zero_rate", nzero, 0, artdaq::MetricMode::Rate);
+    sbndaq::sendMetric(group_name, fragment_id, "frag_count", frag_count, 0, artdaq::MetricMode::Average);
+    sbndaq::sendMetric(group_name, fragment_id, "zero_rate", nzero, 0, artdaq::MetricMode::Rate);
      
-     }//end loop over handle
-    
-      //End copy from FragmentDQMAna_module.cc
+  }//end loop over handle
 
-    //Concatenate hit vectors from each fragment into an event-long hit vector.
-    hit_vector.insert(hit_vector.end(),this_hit_vector.begin(),this_hit_vector.end());
-  //}//loop over fragment handles ???
+  //Concatenate hit vectors from each fragment into an event-long hit vector.
+  hit_vector.insert(hit_vector.end(),this_hit_vector.begin(),this_hit_vector.end());
   
-    ///////////////////////////////////////
-    // Extract Information from the Hits //
-    ///////////////////////////////////////
+  ///////////////////////////////////////
+  // Extract Information from the Hits //
+  ///////////////////////////////////////
   
   //Event-level variables for art root events - basic checks unnecessary for online monitoring
-  size_t num_fragments = fragmentHandle->size();
-  size_t num_hits = hit_vector.size();
+  //size_t num_fragments = fragmentHandle->size();
+  //size_t num_hits = hit_vector.size();
```

Comment:
remove?

Reply:
if we nuke the fragment metrics happy to! Leaving these be for now :p